auth/auth/auth.py

```diff
@@ -816,6 +816,9 @@ async def get_userinfo(request: web.Request, auth_token: str) -> UserData:
 
     userdata = await get_userinfo_from_hail_session_id(request, auth_token)
     if userdata:
+        db = request.app[AppKeys.DB]
+        await db.just_execute("UPDATE sessions SET created = NOW() WHERE session_id = %s", (auth_token,))
```

Comment:
Ideally we would be more explicit in the naming that "created" now really means "last accessed", but to avoid a database migration, I think it's ok for now